keps/NNNN-kep-template/README.md

```diff
@@ -44,6 +44,10 @@ When editing KEPS, aim for tightly-scoped, single-topic PRs to keep discussions
 focused. If you disagree with what is already in a document, open a new PR
 with suggested changes.
 
+Wrap long lines instead of using one line per paragraph. When updating, change
+as few lines as possible. This helps with anchoring review comments.
```

Comment:
> When updating, change as few lines as possible.

This seems like un-actionable advice. How do we want people to do this?
Naive line wrapping will result in updating _more_ lines per update.

Reply:
How about:

"""
Long lines should be wrapped at around 80 characters, for readability and easier diffing.  Do not use one line per paragraph. When updating a KEP, try to change change as few lines as possible. This helps with anchoring review comments.  Line-wrapping may modify subsequent lines, which is OK. Consider using semantic line breaks (https://sembr.org/).
"""